CHANGELOG.md

```diff
@@ -1,4 +1,93 @@
 
+<!-- 8.1.0-beta1 START -->
+
+# 8.1.0-beta1 (2021-07-22)
+
+### Features and enhancements
+
+* **Alerting:** Add alertmanager notifications tab. [#35759](https://github.com/grafana/grafana/pull/35759), [@nathanrodman](https://github.com/nathanrodman)
+* **Alerting:** add button to deactivate current alertmanager configuration. [#36951](https://github.com/grafana/grafana/pull/36951), [@domasx2](https://github.com/domasx2)
+* **Alerting:** add toggle in Loki/Prom datasource config to opt out of alerting ui. [#36552](https://github.com/grafana/grafana/pull/36552), [@domasx2](https://github.com/domasx2)
+* **Alerting:** allow  any "evaluate for" value >=0 in the alert rule form. [#35807](https://github.com/grafana/grafana/pull/35807), [@domasx2](https://github.com/domasx2)
+* **Alerting:** if cortex alertmanager returns empty user config, load default config from status endpoint. [#35769](https://github.com/grafana/grafana/pull/35769), [@domasx2](https://github.com/domasx2)
+* **Alerting:** view to display alert rule and its underlying data. [#35546](https://github.com/grafana/grafana/pull/35546), [@mckn](https://github.com/mckn)
```

Comment:
I do not know what this means from the title without digging. Moving on to other titles for the time being. Note that I am making changes upstream.